src/items/traits.md

```diff
@@ -10,7 +10,7 @@ interface consists of [associated items], which come in three varieties:
 All traits define an implicit type parameter `Self` that refers to "the type
 that is implementing this interface". Traits may also contain additional type
 parameters. These type parameters (including `Self`) may be constrained by
-other traits and so forth as usual.
+other traits and so forth [as usual].
```

Comment:
Purely a style nit so feel free to disagree: would it make sense to have [as usual][generics] so it's more clear to someone reading the source where pothole links are going?

Reply:
I'm heading to work now, so I'll change it when I get home.